### PR TITLE
chore: use CircleCI for publish

### PR DESCRIPTION
**Pull request by JustinBeckwith**

### CODE REVIEW

**Comment by theacodes** [comment]
Can we not do this and set up a kokoro job instead?

File: .circleci/config.yml [hunk 1]
```diff
@@ -12,64 +12,34 @@ workflows:
...
-          requires:
-            - node8
-            - node10
-          filters:
-            tags:
-              only: /.*/
-
+            branches:
+              ignore: /.*/
+            tags: *releases
```
Comment (alexander-fenster) [inline .circleci/config.yml]: `*releases` reference is undefined.

```
Build-agent version 0.0.7546-3d4d9eb8 (2018-07-20T14:36:52+0000)
Configuration errors: 2 errors occurred:

* Error parsing config file: yaml: unknown anchor 'releases' referenced
* Cannot find a job named `build` to run in the `jobs:` section of your configuration file.
If you expected a workflow to run, check your config contains a top-level key called 'workflows:'
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -12,64 +12,35 @@ workflows:
...
-      - image: node:10
+      - image: 'node:8'
         user: node
     steps:
       - checkout
-      - run: *npm_install
-      - run:
-          name: Build documentation.
-          command: npm run docs
+      - run: echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
```
Comment (alexander-fenster) [inline .circleci/config.yml]: The variable is not set. I'll add one now.

Comment (alexander-fenster) [reply .circleci/config.yml]: Added `NPM_TOKEN`

**Comment by JustinBeckwith** [comment]
Sure! Do you want to take a shot, or should we wait for you to pair up with @kinwa91?

**Comment by theacodes** [comment]
It's been ready. We have extensive docs on the dpe team site for setting up
kokoro. I'm happy to review and help if folks get stuck.

**Comment by alexander-fenster** [comment]
@JustinBeckwith can we try Kokoro then? It might be a good small and non-customer-facing project to start with.

**Comment by JustinBeckwith** [comment]
Sure :) @kinwa91 is taking the lead on this for the nodejs side next week.  For now I can just #yolo release this.

**Pull request merged**